core/src/main/scala/org/apache/spark/internal/io/HadoopMapReduceCommitProtocol.scala

```diff
@@ -99,7 +99,7 @@ class HadoopMapReduceCommitProtocol(jobId: String, path: String)
   }
 
   private def getFilename(taskContext: TaskAttemptContext, ext: String): String = {
-    // The file name looks like part-r-00000-2dd664f9-d2c4-4ffe-878f-c6c70c1fb0cb_00003.gz.parquet
+    // The file name looks like part-00000-2dd664f9-d2c4-4ffe-878f-c6c70c1fb0cb_00003.gz.parquet
```

Comment:
After more reading, the exist hive table writers do not have such an issue. It is based on a unique ID [`TaskAttemptID`](https://hadoop.apache.org/docs/r2.6.3/api/org/apache/hadoop/mapreduce/TaskAttemptID.html), which is generated by the function call of `FileOutputFormat.getTaskOutputPath`